guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -201,6 +201,27 @@ endif::[]
 # journalctl -u leapp_resume -f
 ----
 
+ifdef::satellite[]
+. Complete the post-upgrade steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+. If you require SELinux to be in enforcing mode, in addition to completing the steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/applying-security-policies_upgrading-from-rhel-7-to-rhel-8#changing-selinux-mode-to-enforcing_applying-security-policies[Changing SELinux mode to enforcing] in the _Upgrading from RHEL 7 to RHEL 8_ guide, complete these additional steps:
++
+----
+# foreman-maintain service stop
+----
++
+----
+# dnf reinstall foreman-selinux katello-selinux --disableplugin=foreman-protector
+----
++
+----
+# foreman-maintain service start
+----
++
+----
+# sleep 20 && hammer ping
+----
```

Comment:
```suggestion
----
# dnf reinstall foreman-selinux katello-selinux --disableplugin=foreman-protector
----
```

This is the only step that is *really* required, if you perform it *before* following the rest of the "enable selinux again" guide